.github/workflows/coverage.yml

```diff
@@ -55,18 +55,30 @@ jobs:
         pip install --progress-bar off .[optional]
         pip install --progress-bar off .[integration] --extra-index-url https://download.pytorch.org/whl/cpu
 
+        echo 'import coverage; coverage.process_startup()' > sitecustomize.py
+
     - name: Tests
       env:
         OMP_NUM_THREADS: 1
+        PYTHONPATH: .
+        COVERAGE_PROCESS_START: .coveragerc
+        COVERAGE_COVERAGE: yes
```

Comment:
```suggestion
        COVERAGE_COVERAGE: yes  # https://github.com/nedbat/coveragepy/blob/65bf33fc03209ffb01bbbc0d900017614645ee7a/coverage/control.py#L255-L261
```

Reply:
Looks good! I added some comments to them in abe39b7 in addition to your suggestion.